javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -254,6 +254,20 @@ object Character {
   @inline def toCodePoint(high: scala.Char, low: scala.Char): Int =
     ((high & SurrogateUsefulPartMask) << 10) + (low & SurrogateUsefulPartMask) + 0x10000
 
+  def codePointCount(seq: CharSequence, beginIndex: Int, endIndex: Int): Int = {
+    if (endIndex > seq.length() || beginIndex < 0 || endIndex < beginIndex)
+      throw new IndexOutOfBoundsException
+    var res = endIndex - beginIndex
+    var i = beginIndex
+    val end = endIndex - 1
+    while (i < end) {
+      if (Character.isSurrogatePair(seq.charAt(i), seq.charAt(i + 1)))
```

Comment:
Is it intentional that this call is qualified with `Character`? We are *in* `Character` here.

Reply:
mostly just because I wanted to show the straight-move/no changes for the original code.  Happy to update this though!

Reply:
updated/removed